tox.ini

```diff
@@ -0,0 +1,47 @@
+# This configuration file is for Tox. https://tox.readthedocs.io/
+# It houses configuration sections for common Python tools, too.
+
+[tox]
+envlist =
+    flake8
+    pylint
+    readme
+    py{27,34,35,36}-django111
+    py{34,35,36}-django20
+
+[testenv]
+commands =
+    flake8: flake8
+    pylint: pylint --rcfile=tox.ini pylint_django setup
+    readme: python setup.py check --restructuredtext --strict
+    py{27,34,35,36}-django{111,20}: coverage run test/test_func.py
+    clean: find . -type f -name '*.pyc' -delete
+    clean: find . -type d -name __pycache__ -delete
+    clean: rm -rf build/ .cache/ dist/ .eggs/ pylint_django.egg-info/ .tox/
+deps =
+    flake8: flake8
+    pylint: pylint
+    pylint: Django>=2.0,<2.1
+    readme: readme_renderer
+    django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
+    py{27,34,35,36}: coverage
+    py{27,34,35,36}: djangorestframework
+    py{27,34,35,36}: psycopg2
+    py{27,34,35,36}: django-tables2
+    py{27,34,35,36}: pylint-plugin-utils
+    py{27,34,35,36}: pytest
```

Comment:
Is it possible to add all of these into a requirements.txt (for devel) file and specify that in tox.ini ?

Reply:
I don't think so. These are conditional dependencies. I think it's possible to use `-rrequirement.txt`, but we would then probably have to create a single requirements file for each condition.

Reply:
As it is now all of these dependencies are installed unconditionally in .travis.yml except accounting for the Django version. I'm fine to continue to do so because these dependencies are only used during testing.